src/v2/Apps/Artist/Routes/Articles/__tests__/Articles.jest.tsx

```diff
@@ -8,6 +8,7 @@ import { graphql } from "react-relay"
 import { Breakpoint } from "v2/Utils/Responsive"
 
 jest.unmock("react-relay")
+jest.mock("v2/Components/Pagination/useComputeHref")
```

Comment:
I moved the href logic into its own hook because its much easier to mock this way, versus other pagination weirdness (and the fact that theres various pagination tests spanning a few different components)